index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <nav id="navbar">
+        <header>
+          <h1>JS Documentation</h1>
+        </header>
+        <div class="nav-wrapper">
```

Comment:
Please use the <nav> semantic tag here instead.

Reply:
To be more specific here, which semantic tag should I make use of. eg. I should make use of the `<nav>` tag instead of the `<div>` tag. semantic tags help us build better site structures.